packages/react/src/components/ComposedModal/ComposedModal.jsx

```diff
@@ -210,6 +237,8 @@ class ComposedModal extends React.Component {
               // Prevent double scrollbars
               [`${iotPrefix}--composed-modal__body--small-margin-bottom`]: error,
             })}
+            // subtracting 4rem for padding/margin
+            style={{ [isLarge ? 'height' : 'maxHeight']: `calc(${maxContentHeight}px - 3rem)` }}
```

Comment:
On the other hand, if we only set `maxHeight` it will fix the double scrollbar problem, but we still need a CSS sizes solution and add those to the individual specialised components (SelectUsersModal, TableColumnCustomizationModal etc)